File: OutBoundManager/src/sunin/outbound/model/define/ConstBatch.java
package sunin.outbound.model.define;

public class ConstBatch {
	public final static String TableName_Real = "BATCH";
	public final static String TableName = "Batch";
	public final static String batchId = "BATCHID";
	public final static String taskId = "TASKID";
	public final static String batchName = "BATCHNAME";
	public final static String batchState = "BATCHSTATE";

	// 当前状态：运行、停止、删除
	public final static String batchState_Run = "Run";
	public final static String batchState_Stop = "Stop";
	public final static String batchState_Delete = "Delete";
}

File: OutBoundManager/WebContent/_global/js/tools/_Const_Batch.js
function _Const_Batch() {

}

// 当前状态
// 执行
_Const_Batch._BatchState_Run = "Run";
// 停止
_Const_Batch._BatchState_Stop = "Stop";
// 删除
_Const_Batch._BatchState_Delete = "Delete";

_Const_Batch._getBatchStateName = function(batchState) {
	if (_Const_Batch._BatchState_Run.toLowerCase() == batchState.toLowerCase()) {
		return "运行";
	} else if (_Const_Batch._BatchState_Stop.toLowerCase() == batchState.toLowerCase()) {
		return "停止";
	} else if (_Const_Batch._BatchState_Delete.toLowerCase() == batchState.toLowerCase()) {
		return "删除";
	} else {
		return batchState;
	}
};
